server/services/premisesService.ts

```diff
@@ -18,15 +19,14 @@ export default class PremisesService {
     const premisesClient = this.premisesClientFactory(token)
     const premises = await premisesClient.all(selectedAreaId)
 
-    return premises.sort((a, b) => {
-      if (a.name < b.name) {
-        return -1
-      }
-      if (a.name > b.name) {
-        return 1
-      }
-      return 0
-    })
+    return premises.sort((a, b) => (a.name < b.name ? -1 : 1))
+  }
+
+  async getCas1All(token: string, gender: ManWoman = null): Promise<Array<Cas1PremisesSummary>> {
+    const premisesClient = this.premisesClientFactory(token)
+    const premises = await premisesClient.allCas1(gender)
+
+    return premises.sort((a, b) => (a.name < b.name ? -1 : 1))
```

Comment:
`localeCompare` might be more robust for sorting strings:

```suggestion
    return premises.sort((a, b) => a.name.localeCompare(b.name))
```

Reply:
True, would be better if the strings may contain numerics.